pkgs/development/interpreters/python/cpython/3.5/default.nix

```diff
@@ -89,6 +89,9 @@ in stdenv.mkDerivation {
 
     # Python on Nix is not manylinux1 compatible. https://github.com/NixOS/nixpkgs/issues/18484
     echo "manylinux1_compatible=False" >> $out/lib/${libPrefix}/_manylinux.py
+
+    # Use Python3 as default python
+    ln -s "$out/bin/python3" "$out/bin/python"
```

Comment:
The `python-wip` branch is meant to sniff out breakage caused by Python3 as default on Hydra, so I've added this to ensure `patchShebangs` works. In the Python3-as-default future (where the default install doesn't include Python2), we'll need this to provide a `python` executable. As I mentioned in the issue, we should also stop `python2` from installing a `python` executable to allow simultaneous installations, but IMO that can wait as well (shouldn't affect Hydra results).

Confusion with any change like this is inevitable; I think the best way to address it is to wait until next release to do the actual switchover, and to clearly mark this as a breaking change in the release notes.